auto_round/autoround.py

```diff
@@ -891,7 +891,7 @@ def quant_block(self, block, input_ids, input_others, q_input=None, device=torch
                 output_q = block_forward(
                     block, current_input_ids, current_input_others, self.amp, self.amp_dtype, device
                 )
-                if self.amp and not check_is_cpu(device):
+                if self.amp:
```

Comment:
amp should be false if cpu does not support bf16

Reply:
> amp should be false if cpu does not support bf16

this is done in the set_amp_dtype()